### PR TITLE
Improve Map Accessibility with Local Language Translations and Set Max Zoom Level

### PR DESCRIPTION
**Pull request by hashim-cpro**
This pull request enhances the user experience and accessibility of the map by:

## Adding Tile Layer with Local Translations:

Replaced the previous tile layer with one that includes translations for local scripts, especially in regions like China, Afghanistan, and Pakistan. This resolves the issue of untranslated labels that made navigation challenging for users unfamiliar with certain scripts.
## Setting Maximum Zoom-Out Level:

Added a maximum zoom level to prevent excessive zooming out, ensuring the map view remains within practical bounds for better usability.
## Issue Resolved:
- #11 

# Screenshots:
# Before: 
image
image
image

# After: 
English translation with local languages
image
max-zoom level
image
max-zoom

### CODE REVIEW

File: club-map.js [hunk 1]
```diff
@@ -1,76 +1,88 @@
...
-      }).addTo(map);
+  const map = L.map(el, {
+    drawControl: true,
+    zoomControl: false, // Disable default zoom control
+    center: [30.683, 9.099], // Set initial center point
+    zoom: 2, // Set an initial zoom level
+    minZoom: 2, // Set minimum zoom-out level
+    maxBounds: [
+      [-90, -180], // Southwest corner of the bounding box
+      [90, 180], // Northeast corner of the bounding box
```
Comment (JosiasAurel) [inline club-map.js:21]: why this addition?

File: club-map.js [hunk 2]
```diff
@@ -1,76 +1,88 @@
...
+    ],
+  });
+
+  L.control.zoom({ position: "topright" }).addTo(map);
+
+  L.tileLayer("https://tile.openstreetmap.de/{z}/{x}/{y}.png", {
```
Comment (JosiasAurel) [inline club-map.js:27]: what does the change on this line do?

**Review by JosiasAurel — CHANGES_REQUESTED** [review]

File: club-map.js (hunk 1 above)
Comment (hashim-cpro) [reply club-map.js:21]: It is used to set/restrict the viewable area of the map, ensuring users can only pan within the specified geographic bounds.

File: club-map.js (hunk 2 above)
Comment (hashim-cpro) [reply club-map.js:27]: It is used to set the tile layer. The one which contains the translations for non-english scriptures.

**Comment by hashim-cpro** [comment]
Do you still want me to un-format the changes?

**Review by JosiasAurel — APPROVED** [review]
lgtm

**Pull request merged**